espnet2/asr/espnet_model.py

```diff
@@ -59,17 +59,25 @@ def __init__(
         report_wer: bool = True,
         sym_space: str = "<space>",
         sym_blank: str = "<blank>",
+        sym_sos: str = "<sos/eos>",
```

Comment:
Is it only used for this HF transformer decoder?
If so, it is better to add some comments.

Reply:
It is used for any decoder (in `_calc_att_loss`), but the default values are set as in a usual ESPnet recipe. HF Transformers need to customize these values according to pretrained tokenizer. Should I add a comment about this?

Reply:
Oh, OK.
Yes, it would be great if you add some comments about them.

Reply:
Done.